src/transaction/log_recovery_context.cpp

```diff
@@ -116,3 +116,9 @@ log_recovery_context::get_end_redo_lsa () const
 {
   return m_end_redo_lsa;
 }
+
+bool
+log_recovery_context::is_page_server () const
+{
+  return m_is_page_server;
```

Comment:
`m_is_page_server` can be `const`'ed.

Reply:
its value is initialized during construction.

```
log_recovery_context.cpp(26,3): error C2166: l-value specifies const object
```

Reply:
Yes, I knew. Should have been moved to initializer list after `const`. :)